src/scroll-animation.test.js

```diff
@@ -228,6 +228,27 @@ describe("ScrollAnimation -", function () {
       });
   });
 
+  it("only animates once with 'animateOnce' property", (done) => {
+    var scrollAnimation = createScrollAnimationOffScreen({animateIn: "zoomIn", animateOnce: true });
+    scrollIntoCompleteView(scrollAnimation);
+    waitFor(() => {return scrollAnimation.node.className.includes("zoomIn")},
+      () => {
+        expect(scrollAnimation.node.className).toContain("zoomIn");
+        scrollToBottom();
+        waitFor(() => {return !scrollAnimation.node.className.includes("zoomIn")},
```

Comment:
At this point I think there's some confusion. The animations are fired by adding the "zoomIn" class. Without animateOnce, when we scroll away from the element the "zoomIn" class is removed, so that we can then add it again when we scroll back to the element, in order to trigger the animation a second time.

With your addition, if animateOnce is set we don't remove the "zoomIn" class when scrolling away from the element, which is fine as we don't intend to add it again to restart the animation. Your test is checking that we do remove the "zoomIn" class though, and only passes because we never actually set the animateOnce property in the createScrollAnimationOffScreen function :)

I've fixed it up a bit and will merge my changes after merging your pull request.